### PR TITLE
Fix bug in get() of nn.Module 

### PR DESCRIPTION
**Pull request by sukhadj**
## Description
When a model with particular layers (BatchNorm2D) is sent to a worker, it produces a KeyError when `copy()` is called on them.

To reproduce:
```python
class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()
        self.l = nn.Linear(10,10)
        self.bn1 = nn.BatchNorm2d(64)
 
model  = Model()         
model_copy = model.copy()  #works fine

model.send(alice)
model.get()

model.copy()   #throws KeyError Exception

```
This is due we were not retrieving buffers along with parameters.

## Type of change

Please mark the options that are relevant.

- [ ] Added/Modified tutorials
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

* [x] I have added tests for my changes
* [x] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
* [x] I have commented my code following [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

### CODE REVIEW

**Comment by LaRiffle** [comment]
I guess this is a duplicate with https://github.com/OpenMined/PySyft/pull/3472 :/

**Comment by sukhadj** [comment]
> I guess this is a duplicate with #3472 :/

Ohh!! I should have checked the open PRs and issues. :sweat_smile: 
That PR solves this issue hence closing!!
